apps/examples/testcase/ta_tc/systemio/itc/itc_uart.c

```diff
@@ -270,19 +322,37 @@ void itc_systemio_iotbus_uart_set_flowcontrol_p(void)
 void itc_systemio_iotbus_uart_write_read_p(void)
 {
 	int ret = IOTBUS_ERROR_NONE;
-	char sz_input_text[BUF_LEN] = "UART READ/WRITE ITC TESTING!";
+	char sz_input_text[BUF_LEN] = "UART R/W TEST";
 	char sz_output_text[BUF_LEN];
 	iotbus_uart_context_h h_uart = iotbus_uart_init(DEVPATH);
 	TC_ASSERT_NEQ("iotbus_uart_init", h_uart, NULL);
 
+	// turn on loopback mode
```

Comment:
Why do we need loopback setting in all these api's?
There is a separate API for loopback test. All these apis are supposed to be normal uart test without loopback. If we set loopback options in all these test cases, then how will they work for boards which do not have uart loopback?

Reply:
`Why do we need loopback setting in all these api's?`
uart write, read TCs will pass only if loopback mode is supported.

`All these apis are supposed to be normal uart test without loopback`
No. without loopback,how can we test write and read? so loopback needs to be enabled, wherever write and read is used.

`how will they work for boards which do not have uart loopback?`
if loopback mode is not supported by any board then we can not test write read on that board.

Reply:
Ok. So, if there is no way to test uart without loopback, then these changes look good. 
But, in that case, we dont need separate api called `itc_systemio_iotbus_uart_loopback_p` because we are testing the exact same functionality in this api `itc_systemio_iotbus_uart_write_read_p`

Reply:
done